src/plugins/hetero/src/plugin.cpp

```diff
@@ -0,0 +1,289 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+#include <memory>
+#include <vector>
+#include <map>
+#include <string>
+#include <utility>
+#include <fstream>
+#include <unordered_set>
+
+// LEGACY
+#include "ie_metric_helpers.hpp"
+// LEGACY
+
+#include "openvino/runtime/device_id_parser.hpp"
+#include "openvino/runtime/properties.hpp"
+#include "openvino/runtime/internal_properties.hpp"
+#include "openvino/util/common_util.hpp"
+
+#include "plugin.hpp"
+// #include "compiled_model.hpp"
+#include "itt.hpp"
+// #include "properties.hpp"
+
+// TODO (vurusovs) required for conversion to legacy API 1.0
+#include "converter_utils.hpp"
+#include "../plugin.hpp"
+#include "../executable_network.hpp"
+#include "../internal_properties.hpp"
+
+#include "iplugin_wrapper.hpp"
+
+namespace {
+HeteroPlugin::Configs any_copy(const ov::AnyMap& params) {
```

Comment:
Let's use ov::AnyMap instead of legacy representation

Reply:
Done